gradle/shipkit.gradle

```diff
@@ -10,3 +10,9 @@ afterEvaluate {
     //TODO let's create a property for this setting in 'shipkit' extension
     tasks.updateReleaseNotes.publicationRepository = "https://plugins.gradle.org/plugin/org.shipkit.java"
 }
+
+apply plugin: 'org.shipkit.upgrade-downstream'
+
+upgradeDownstream{
+    repositories = ['mockito/shipkit-example']
```

Comment:
Let's add 'shipkit-bootstrap', too.

Later on if this works well for us, let's add Mockito. This will be awesome!!! Thanks a lot for working on this plugin!